AnalysisReporting/analysisreport/templates/analysisreport/landingpage.html

```diff
@@ -0,0 +1,85 @@
+{% load static%}
+<!DOCTYPE html>
+<html>
+<head>
+	<title>login</title>
+
+	<meta charset="utf-8">
+    <title> Signup </title>
```

Comment:
There are two title tags in the header section, keep only one.  
Wrong title given to landing page, change it.